pinot-core/src/main/java/org/apache/pinot/core/realtime/stream/StreamMessageMetadata.java

```diff
@@ -18,24 +18,22 @@
  */
 package org.apache.pinot.core.realtime.stream;
 
-import org.apache.pinot.annotations.InterfaceAudience;
-import org.apache.pinot.annotations.InterfaceStability;
+import org.apache.pinot.common.metadata.RowMetadata;
 
 
 /**
  * A class that provides metadata associated with the message of a stream, for e.g.,
  * ingestion-timestamp of the message.
  */
-@InterfaceAudience.Public
-@InterfaceStability.Evolving
-public class StreamMessageMetadata {
+public class StreamMessageMetadata implements RowMetadata {
```

Comment:
Let's rename this to DefaultRowMetadata or StreamRowMetadata? In the future we might want to make this class support more types of metadata

Reply:
I think StreamMessageMetadata works better (we use messages in the context of streams).